styles/globals.css

```
@@ -22,12 +22,15 @@ h5,
 h6 {
   @apply tracking-tight;
 }
-
-input[type="checkbox"]:focus-visible,
-[role="checkbox"]:focus-visible,
-[role="radio"]:focus-visible,
-[role="combobox"]:focus-visible,
-[role="switch"]:focus-visible,
+button:not([role]){
+  @apply border;
+  @apply border-transparent;
+}
```

nit: add spacing here
```suggestion
}

```